openstudiocore/src/model/AirLoopHVAC.cpp

```diff
@@ -914,6 +914,25 @@ namespace detail {
     bool result = setString(OS_AirLoopHVACFields::DesignSupplyAirFlowRate, "autosize");
     OS_ASSERT(result);
   }
+  
+  /** Baseline Pressure Drop Adjustment **/
+  // Return the Baseline Pdrop
+  double AirLoopHVAC_Impl::baselinePressureDropAdjustment() const {
+    return getDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment,true);
+  }
+  
+  // Set the Baseline Pdrop
+  bool AirLoopHVAC_Impl::setBaselinePressureDropAdjustment(double baselinePressureDropAdjustment) {
+    result = setDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment, baselinePressureDropAdjustment);
+    OS_ASSERT(result);
```

Comment:
Here you don't want the assert, result could be true or false (e.g. if user passes in -9999), we return the value to the user and don't really care which way it goes.  You do need to return a value or it won't compile, also you have to declare the result variable:

    bool result = setDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment, baselinePressureDropAdjustment);
    return result;